js/index.js

```diff
@@ -635,6 +659,20 @@ $(function() {
 
                           myconsole.warn("Unable to load JS bundle", err);
                         });
+                      } else if ($('input[name=backend_inputs]').filter(':checked').val() === "mathbox") {
+                           $.when($.get("js/mathbox-bundle.js"),
+                                  $.get(backend.endpoint + "/bundle")
+                                ).done(function(mathboxBundle, bundle) {
+
+                           var replaced = bundle[0].replace(/require\("Data.Either"\)/g, "PS['Data.Either']")
+                                                   .replace(/require\("Data.Maybe"\)/g, "PS['Data.Maybe']")
+                                                   .replace(/require\("Mathbox.Field"\)/g, "PS['Mathbox.Field']")
```

Comment:
Not sure why this is needed exactly. 

When similar examples are created [here](https://github.com/rintcius/purescript-mathbox/blob/master/build-examples.sh) then the bundle works as-is.